.github/workflows/tnf-image.yaml

```diff
@@ -23,7 +23,7 @@ env:
   IMAGE_TAG: latest
   TNF_CONTAINER_CLIENT: docker
   TNF_NON_INTRUSIVE_ONLY: false
-  TNF_MINIKUBE_ONLY: true
```

Comment:
is it still used ?

Reply:
Whoops no, I was messing with this prior.  I'll fix.